test/README.md

```diff
@@ -75,3 +75,8 @@ There is also a python script that can be run to check mod_proxy_cluster. You ca
 as `miniserver.py`. Execute it simply as `./includes/miniserver.py <port-number>`.
 
 You must have `httpd` with `mod_proxy_cluster` running.
+
+# Testing the lbmethod and mod_proxy_balancer
+Just run the test with CONF=httpd/mod_lbmethod_cluster.conf
+```
```

Comment:
Missing closing ```.

Reply:
This is still relevant (it displays properly only because its at the end of file)-

Reply:
https://github.com/jfclere/mod_proxy_cluster/blob/master/test/README.md?plain=1#L83
It is there, what do I miss?

Reply:
WTH, I must be blind! I'm sorry, you're right.